apis/stardust/api-spec/spec.json

```diff
@@ -720,6 +720,11 @@
           "optional": true,
           "defaultValue": false,
           "type": "boolean"
+        },
+        "dataViewType": {
```

Comment:
Maybe want to explicitly state that the type should be an `sn-table` but might be registered under a different name? I guess they techincally could set any object as chart here but since we pass in the specific `sn-table` properties when creating the view data object I assume that is not really supported?

Reply:
This can be overwritten when you set this in your initial config. So if you do not register something as `sn-table`, but just as table you would need to add that here as well

Reply:
I'm just saying that we want to clearly document what the expections are. The object should be an `sn-table` but might be registered as something else, ex "table". Also assuming that the consumer of the `toggleDataView` API must have registered the `sn-table` (or by some other name) for it to work. Which is also something that should be documented IMO

Reply:
Got it. I added some more explanation now, hopefully that makes since more clear